frontend/src/store/index.js

```diff
@@ -252,25 +263,34 @@ export const store = new Vuex.Store({
   getters: {
     room_resources(state) {
       if (state.rooms.length > 0) {
-        return state.rooms.map(room =>
+        let rooms = state.rooms.map(room =>
           ({
             id: room.room_id,
             title: room.room_name,
             eventColor: room.color
           })
         )
+        rooms.push({
+          id: 2,
+          title: 'blah',
+          eventColor: 'green'
+        })
+        return rooms
       }
       return []
     },
     
     calendar_events(state) {
       if (state.bookings.length > 0) {
+        function changeTime(time) {
+          return time.split('+')[0] + '-08:00'
```

Comment:
Does this imply that the server is not returning properly time zoned objects?

Reply:
yeah this likely is not needed.  I wrote this for testing when i was handling the bootstrapped data which was all in UTC.  The posts to /calendar are all localized by the browser to -08:00.  hopefully its accepting that properly and returning as posted.  i'll get back to you after i remove that changeTime function and see if i'm still getting correctly formatted time.